tdvt/tdvt/tdvt.py

```diff
@@ -102,12 +97,12 @@ def copy_files_to_zip(self, dst_file_name, src_dir, is_logs):
         optional_dir_name = self.test_config.config_file.replace('.', '_')
         if is_logs is True:
             log_dir = os.path.join(src_dir, optional_dir_name)
-            glob_path = glob.glob(os.path.join(log_dir, 'log*.txt'))
-            glob_path.extend(glob.glob(os.path.join(log_dir, 'tabprotosrv*.txt')))
+            glob_path = glob.glob(os.path.join(log_dir, '*.txt'))
+            glob_path.extend(glob.glob(os.path.join(log_dir, '*.log')))
             glob_path.extend(glob.glob(os.path.join(log_dir, 'crashdumps/*')))
         else:
             glob_path = glob.glob(os.path.join(src_dir, 'actual.*'))
-        with ZipFile(dst, mode) as myzip:
+        with zipfile.ZipFile(dst, mode, zipfile.ZIP_DEFLATED) as myzip:
```

Comment:
From [the Python docs](https://docs.python.org/3/library/zlib.html#module-zlib), I see that `zipfile.ZIP_DEFLATED` uses the `zlib` module, which in turn has an external dependency on `zlib`. Don't think that'll be an issue, but just something to have in the back of our heads if anyone complains about it.

Reply:
I was a bit concerned about that too. I tested on windows, mac and linux and we at least have it installed there. If we have problems we could do a try/catch with the import statement and either compress or not based on that.